pkg/auth/server/headers/headers.go

```diff
@@ -0,0 +1,28 @@
+package headers
+
+import (
+	"net/http"
+)
+
+// We cannot set HSTS by default, it has too many drawbacks in environments
+// that use self-signed certs
+const standardHeaders map[string]string{
+	// Turn off caching, it never makes sense for authorization pages
+	"Cache-Control": "no-cache, no-store",
+	"Pragma": "no-cache",
+	"Expires": "0",
+	// Use a reasonably strict Referer policy by default
+	"Referrer-Policy": "strict-origin-when-cross-origin",
+	// Do not allow embedding as that can lead to clickjacking attacks
+	"X-Frame-Options": "DENY",
+	// Add other basic scurity hygiene headers
+	"X-Content-Type-Options": "nosniff",
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options ``nosniff`` could be troublesome if applied to image resources. Are default header only applied to HTML sources?

> Note: nosniff only applies to "script" and "style" types. Also applying nosniff to images turned out to be incompatible with existing web sites.

Reply:
We do not send images in OSIN afaik, but also that page says:
"Note: nosniff only applies to "script" and "style" types." so I don't think we have a problem.